lisa/tests/base.py

```diff
@@ -903,13 +934,25 @@ def rtapp_profile(self):
         return self.get_rtapp_profile(self.plat_info)
 
     @property
-    def rtapp_tasks(self):
+    def rtapp_tasks_from_profile(self):
         """
         Sorted list of rtapp task names, as defined in ``rtapp_profile``
         attribute.
         """
         return sorted(self.rtapp_profile.keys())
 
+    @property
+    def rtapp_tasks(self):
```

Comment:
Looks like there is no value in having `get_trace_rtapp_tasks()` separated from the property, so you can just rename `get_trace_rtapp_tasks()` into:
```
@property
@memoized
def rtapp_tasks(self):
  # body of get_trace_rtapp_tasks()
```

Reply:
That's because I forgot to force call it when we go `bundle.trace` directly... fix coming soon.